tools/bin/ariaDownload.py

```diff
@@ -307,6 +318,32 @@ def check_cookie_is_logged_in(cj):
             return True
     return False
 
+def rewrite_summary(infos):
+    succeed, failed, skipped, tot_time, tot_bytes = [], [], [], 0, 0
+    for info in infos:
+        if info[0]: succeed.extend(info[0])
+        if info[1]: failed.extend(info[1])
+        if info[2]: skipped.extend(info[2])
+        tot_time += float(info[3])
+        tot_bytes += float(info[4])
+
+
+    log.info ('Successes: %d files, %s bytes', len(succeed), tot_bytes)
+    [print ('\n\t-', sf['file'], f'{sf["size"]/1024**2:.2f}') for sf in succeed]
+
+    if skipped: log.info('Skipped: %d files', len(skipped))
+    [print ('\n\t-', sf) for sf in skipped]
```

Comment:
Codacy found an issue: [Expression "[print('\n\t-', sf) for sf in skipped]" is assigned to nothing](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6310391)